experiments/simbricks/experiment/experiment_environment.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 Max Planck Institute for Software Systems, and
```

Comment:
Fine for this PR, but here again I would in the future do this as separate commits. 1 just to move the class out (and add the necessary imports elsewhere. Then 1 for reformatting.  and one to actually add the annotation.

May sound a bit silly but in general a good habit to get into as most open source projects and many companies with proper code review will strictly require this. Just makes it easier to review, read through history, and do things like git bisect as well. Once you're in the habit of committing these things immediately it does not really add significant effort at all.